HHS/OPRE-OPS
# feat: api integration for summary cards

## What changed

Use API data for frontend CAN features.

## Issue

[2842](https://github.com/HHS/OPRE-OPS/issues/2842)

## How to test

- E2E and unit tests are passing.
- Navigate to the CAN List page and select different fiscal years and filters. 
  - Ensure CAN list and summary cards are updated accordingly. 
- Go to `/cans/504/spending` and select FY 2043. 
  - Ensure budget lines appear in the table above.
  - Ensure the agreement name and procurement shop abbreviation are displayed properly. 
  - Check that summary cards have correct data displayed.

## Screenshots

N/A

## Definition of Done Checklist
<del> - [ ] Form validations updated


## Links

N/A
